Include <fcntl.h> to get loff_t definition

Fixes the following build issue with musl:

dqblk_v2.h:32:2: error: unknown type name ‘loff_t’

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/dqblk_v2.h
===================================================================
--- a/dqblk_v2.h
+++ b/dqblk_v2.h
@@ -8,6 +8,7 @@
 #define GUARD_DQBLK_V2_H
 
 #include <sys/types.h>
+#include <fcntl.h>
 #include "quota_tree.h"
 
 #define Q_V2_GETQUOTA	0x0D00	/* Get limits and usage */
